### PR TITLE
Adding geolocation

### PR DESCRIPTION
**Pull request by cadnce**

### CODE REVIEW

**Review by forseti7 — CHANGES_REQUESTED** [review]
-Change tabs to spaces-

Looks perfect

**Review by stevenjfr — COMMENTED** [review]
Think we may have a bit of a problem with the indentation Matt. please review.

File: frontend/src/features/upload/upload.pug [hunk 1]
```diff
@@ -1,16 +1,18 @@
-h4 Upload Sighting
+script.h4 Upload Sighting
```
Comment (cadnce) [inline frontend/src/features/upload/upload.pug]: Noob

Comment (forseti7) [inline frontend/src/features/upload/upload.pug]: Small issue here Matt

**Review by stevenjfr — APPROVED** [review]
Looks much better. Excellent work. Look forward to seeing it in production! 
Fraz

**Review by forseti7 — APPROVED** [review]

**Comment by forseti7** [comment]
Top work Matt, keep it up

**Comment by cadnce** [comment]
I'm not even convinced this is web anymore!

**Comment by cadnce** [comment]
closes #55

**Pull request merged**